examples/misc/test/library_tour/convert_test.dart

```diff
@@ -56,7 +57,7 @@ void main() {
     assert(funnyWord == 'Îñţérñåţîöñåļîžåţîờñ');
     // #enddocregion utf8-decode
     // #docregion utf8-encode
-    List<int> encoded = utf8.encode('Îñţérñåţîöñåļîžåţîờñ');
+    Uint8List encoded = utf8.encode('Îñţérñåţîöñåļîžåţîờñ');
```

Comment:
If you want to avoid the `import "dart:typed_data";` import, you can also rely on type inference, just making this `var` or `final`.

Looking at some other code in the docs at [0] it seems to use `var` a lot.

[0] https://dart.dev/guides/libraries/library-tour#dartconvert---decoding-and-encoding-json-utf-8-and-more

Reply:
I think either way is fine. Relying on type inference is consistent with the nearby examples and what our docs default to, but showing the type here helps a bit to support the leading text that it returns a "list of UTF8-encoded bytes:".